src/components/Calculator.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+const Calculator = () => (
```

Comment:
-  [ ] In this activity, we want the _Calculator_ component to be class-based component. Kindly improve upon this aspect.